errors/bad-request.js

```diff
@@ -0,0 +1,18 @@
+import BaseError from './base';
+
+function BadRequestError(
+  name = 'BadRequestError',
+  message,
+  type = 'bad_request_error',
+  errors = []
+) {
+  this.statusCode = 400;
+  this.name = name;
+  this.message = message;
+  this.type = type;
+  this.errors = errors;
+}
+
+BadRequestError.prototype = new BaseError();
+
+export default BadRequestError;
```

Comment:
Essa sintaxe com `prototype` não é muito intuitiva, e se a gente usasse classes, num formato bem parecido com a solução no #140 [api-error.js](https://github.com/BrasilAPI/BrasilAPI/blob/b428363c85d4c9ca6f464a3323a34df99a39d3d5/errors/api-error.js), mas separando em objeto apenas os parâmetros opcionais?

```suggestion
export default class BadRequestError extends Error {
  constructor(
    message,
    { name = 'BadRequestError', type = 'bad_request_error', errors = [] }
  ) {
    super(message);

    this.statusCode = 400;
    this.name = name;
    this.message = message;
    this.type = type;
    this.errors = errors;
  }
}
```

Reply:
Corrigido!